### PR TITLE
Fix/side param

### PR DESCRIPTION
**Pull request by PetervDooren**
Merge together with https://github.com/tue-robotics/tue_robocup/pull/1083

### CODE REVIEW

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
Can we get rid of `left` as well?

**Review by MatthijsBurgh — DISMISSED** [review]

File: src/gripper_controller/gripper_controller.py [hunk 1]
```diff
@@ -20,30 +20,20 @@ def __init__(self):
...
-                                                                       GripperCommandAction,
-                                                                       execute_cb=self.gripper_left,
-                                                                       auto_start=False)
-        self.srv_safe_joint_change_right = actionlib.SimpleActionServer('right_arm/gripper/action',
-                                                                        GripperCommandAction,
-                                                                        execute_cb=self.gripper_right,
-                                                                        auto_start=False)
-        self.srv_safe_joint_change_left.start()
-        self.srv_safe_joint_change_right.start()
+        self.srv_safe_joint_change = actionlib.SimpleActionServer('/hero/gripper/action',
```
Comment (MatthijsBurgh) [inline src/gripper_controller/gripper_controller.py]: Why this hardcoded `hero` prefix?

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: src/gripper_controller/gripper_controller.py (hunk 1 above)
Comment (PetervDooren) [reply src/gripper_controller/gripper_controller.py]: Because this entire unit is hero-specific. No need to pretend that it is any more generic than it is

Comment (MatthijsBurgh) [reply src/gripper_controller/gripper_controller.py]: But his is fixed by namespacing the node. Keep it consistent.
```suggestion
        self.srv_safe_joint_change = actionlib.SimpleActionServer('gripper/action',
```

**Review by MatthijsBurgh — DISMISSED** [review]

**Review by jlunenburg — CHANGES_REQUESTED** [review]
Since the gripper controller is hero-specific (it depends on `tmc_manipulation_msgs` and `tmc_control_msgs`), please move it to 'hero_skills' instead of 'robot_skills'.

**Comment by PetervDooren** [comment]
This is the hero_bridge pr. A lot of things must be moved but that is not inn the scope of this pr

**Comment by jlunenburg** [comment]
> This is the hero_bridge pr. A lot of things must be moved but that is not inn the scope of this pr

My apologies, I have mistaken it for the `tue_robocup` one. In that case, I'll approve it based on Matthijs' review (it just caught my eye in my mailbox).

**Review by jlunenburg — DISMISSED** [review]

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**